src/types/index.ts

```diff
@@ -14,3 +14,18 @@ export interface SimplifiedLayout {
   nodes: (ChartNode | TreeNode)[]
   links: any[]
 }
+
+export interface LinkProps {
+  Component: any // TODO types
+  id: string | number
+  d: string
+  quadraticPoint: WithCoords
+  sweep: number
+  length: number
+  label?: string
+  size: number
+  hover: boolean
+  source: ChartNode
+  target: ChartNode
+  onClick: (link: any) => void
```

Comment:
You can change the any with the GraphLink type maybe?
And then move it to the types folder aswell : https://github.com/unirakun/react-kgraph/blob/master/src/Graph.tsx#L46